### ISSUE TITLE
Errors in Historic Downtown Topeka Tour

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/openwig#165]
```
What cartridge are you playing?
http://www.wherigo.com/cartridge/details.aspx?CGUID=58c360c6-f21d-4002-9338-6966
b3daca98

What is your phone/PDA manufacturer and model? What GPS are you using?
Samsung Infuse

What steps will reproduce the problem?
1.Not sure, maybe opening the cartridge, saving progress, and opening cartridge 
again?
2.I received an error stating, visit this Web site and report the issue.
3.


What is the expected output? What happens instead?
I expect to complete the Historic Downtown Topeka Tour; instead it rarely 
"Finishes" "Tasks".

If you see an error screen, please type its contents here:
I have a lot of error information - too much to type here. I could email a log. 
Basically, it shows a bunch of java stuff.

What version are you using?
Not sure. Maybe 0.4?

Please provide any additional information below.

```

Original issue reported on code.google.com by `stingray...@gmail.com` on 30 Nov 2011 at 8:29

**Comment by GoogleCodeExporter** [comment]
```
Yeah, that "bunch of java stuff" is what i need ;e) this site has 
'attachments', could you attach the log file?
```

Original comment by `matej...@gmail.com` on 30 Nov 2011 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
I ended up typing this.  I could not find the logs.

you hit a bug! please report at openwig.googlecode.com and i'll fix it for you!
java.lang.RuntimeException:_add not defined for operands

stack trace: at @E:/
webtempfiles/1d98b6b9-9795-4bdc-9cdb-3129ca8703de/Historic DowntownTopeka 
Tour-new.enc: 1638
at @E:/
webtempfiles/1d98b6b9-9795-4bdc-9cdb-3129ca8703de/Historic DowntownTopeka 
Tour-new.enc: 1616
at @E:/
webtempfiles/1d98b6b9-9795-4bdc-9cdb-3129ca8703de/Historic DowntownTopeka 
Tour-new.enc: 1726
at @E:/
webtempfiles/1d98b6b9-9795-4bdc-9cdb-3129ca8703de/Historic DowntownTopeka 
Tour-new.enc: 1671
```

Original comment by `stingray...@gmail.com` on 30 Nov 2011 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
I updated the issue with the bug information.  I ended up typing the info
out and adding it as a comment.

Let me know if you need any other info.

Thank you!
```

Original comment by `stingray...@gmail.com` on 1 Dec 2011 at 6:03
